.github/workflows/tests.yml

```diff
@@ -14,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["windows-latest", "ubuntu-latest", "macos-latest"]
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
Can we add Python 3.12 as well since we're already mucking with our testing setup?

Reply:
Done